support-frontend/app/controllers/CanonicalLinks.scala

```diff
@@ -6,13 +6,13 @@ trait CanonicalLinks {
 
   def buildCanonicalDigitalSubscriptionLink(countryCode: String, orderIsAGift: Boolean): String =
     if (orderIsAGift) s"${supportUrl}/${countryCode}/subscribe/digital/gift"
-    else s"${supportUrl}/${countryCode}/subscribe/digital"
+    else s"/${countryCode}/subscribe/digital"
```

Comment:
Why has this changed?

Reply:
`supportUrl` is taken from config - [which is set here](https://github.com/guardian/support-frontend/blob/main/support-frontend/conf/PROD.public.conf). 

This essentially makes the link relative the URL, rather than absolute with it in order to support multiple domains.

Reply:
But if the `orderIsAGift` we do use the absolute URL?